src/components/Dashboard/index.js

```diff
@@ -0,0 +1,39 @@
+import LinkExport from './Link';
```

Comment:
Take a look [here](https://github.com/react-bootstrap/react-bootstrap/blob/master/src/index.js) for an example of a more concise way to do this.

Reply:
Updated